.github/workflows/build-and-tests.yml

```diff
@@ -0,0 +1,26 @@
+name: Run build and tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
do we need it to be ubuntu?
maybe 'node:lts-slim' as we used in colosseum would do the trick